misc/helm-charts/operator/README.md

```diff
@@ -346,6 +347,122 @@ kubectl logs -l app.kubernetes.io/name=materialize-operator -n materialize
 
 For more detailed information on using and troubleshooting the Materialize operator, refer to the [Materialize documentation](https://materialize.com/docs).
 
+## Upgrading Materialize
```

Comment:
nit: I'd drop `Materialize` here, and later you can reference `Upgrading Materialize`